mix.exs

```diff
@@ -91,7 +93,10 @@ defmodule TypedStruct.MixProject do
   defp package do
     [
       licenses: ["MIT"],
-      links: %{"GitHub" => @repo_url}
+      links: %{
+        "Changelog" => "https://hexdocs.pm/typed_struct/changelog.html",
```

Comment:
Thanks for the hint, I think this is indeed a good practice to adopt. I’ll also update my coding rules to change this in my project generator and other projects.

Reply:
Indeed. We don't have to go back to GitHub or other hosting repos just to look at the Changelog when we're in HexDocs.